cli/tests/test_advance_bookmarks.rs

```diff
@@ -0,0 +1,455 @@
+// Copyright 2024 The Jujutsu Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::path::Path;
+
+use test_case::test_case;
+
+use crate::common::TestEnvironment;
+
+fn get_log_output_with_bookmarks(test_env: &TestEnvironment, cwd: &Path) -> String {
+    // Don't include commit IDs since they will be different depending on
+    // whether the test runs with `jj commit` or `jj describe` + `jj new`.
+    let template = r#""bookmarks{" ++ local_bookmarks ++ "} desc: " ++ description"#;
+    test_env.jj_cmd_success(cwd, &["log", "-T", template])
+}
+
+fn set_advance_bookmarks(test_env: &TestEnvironment, enabled: bool) {
+    if enabled {
+        test_env.add_config(
+            r#"[experimental-advance-bookmarks]
+        enabled-bookmarks = ["glob:*"]
```

Comment:
maybe these tests are broken now (as we reverted the config name change)?

Reply:
reverted